From f753ee1373a56cee5d6c2e7c107a85822828c983 Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Sat, 18 Dec 2021 16:38:16 +0100
Subject: [PATCH 30/96] dt-bindings: arm: cpus: Document Meson8 TrustZone
 firmware enable-method

Amlogic Meson8 SoCs can run a TrustZone firmware. This results in the
CPU registers not being accessible directly and instead require firmware
calls for booting the secondary cores or powering off a CPU. Add a new
compatible string for this enable-method.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 Documentation/devicetree/bindings/arm/cpus.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/cpus.yaml b/Documentation/devicetree/bindings/arm/cpus.yaml
index cc5a21b47..ec7593e04 100644
--- a/Documentation/devicetree/bindings/arm/cpus.yaml
+++ b/Documentation/devicetree/bindings/arm/cpus.yaml
@@ -216,6 +216,7 @@ properties:
           - allwinner,sun9i-a80-smp
           - allwinner,sun8i-a83t-smp
           - amlogic,meson8-smp
+          - amlogic,meson8-trustzone-firmware-smp
           - amlogic,meson8b-smp
           - arm,realview-smp
           - aspeed,ast2600-smp
-- 
2.45.1

